Formula/aerc.rb

```diff
@@ -0,0 +1,20 @@
+class Aerc < Formula
+  desc "Email client that runs in your terminal"
+  homepage "https://aerc-mail.org/"
+  url "https://git.sr.ht/~sircmpwn/aerc/archive/0.5.1.tar.gz"
+  sha256 "8feb7f87aa210ab7d8b271c34ad39af13f553447994341b3c25e7a659ce37d46"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "scdoc" => :build
+
+  def install
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/aerc", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
it just has `-v` and a `mailto:` argument and that opens the gui... so i only could do a `aerc --help` and that fails with a 
```
2020/11/12 14:51:29 /home/marv/.nix-profile/bin/aerc: unknown option --
2020/11/12 14:51:29 Usage: aerc [-v] [mailto:...]
```

so what should i do?